src/utils/postProcessResults.ts

```diff
@@ -68,11 +68,13 @@ const postProcessResults = (
     })
     .filter((r) => {
       return settings.searchFilter
-        ? Object.values(r).some((value) =>
-            String(value)
-              .toLowerCase()
-              .includes(settings.searchFilter.toLowerCase())
-          )
+        ? Object.keys(r)
+            .filter((key) => !key.endsWith("-uid") || key !== "uid")
```

Comment:
I think you want just `!key.endsWith("-uid")`. Not only is the second condition redundant with the first, the key `foo-uid` will actually return `true` because it'll satisfy the second part of this `||`

Reply:
ah, right.  I changed it to `&&`